server/lobbyconnection.py

```diff
@@ -51,23 +51,25 @@ def __init__(self, message, *args, **kwargs):
 @with_logger
 class LobbyConnection():
     @timed()
-    def __init__(
-        self,
-        games: GameService,
-        players: PlayerService,
-        geoip: GeoIpService,
-        matchmaker_queue: MatchmakerQueue
-    ):
+    def __init__(self,
```

Comment:
Please fix indents to how the were before

Reply:
Lets put the first parameter on a new line and line up the remaining parameters to that (see the deletions in this diff). IMO this is nicer because it creates consistent indentations (even if the function names have different lengths).